PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -94,6 +94,7 @@ CorrectionSigma="${38}"
 RegName="${39}"
 UseIndMean="${40}"
 IsLongitudinal="${41}"
+MetricReg="${42}"
```

Comment:
Currently this argument doesn't get sanity checked anywhere.  If a user passes `--metric-regression=old` or `=OLDE`, it will silently not generate either file.  We may want to sanity check it in PostFS also, so the user gets immediate feedback rather than having some things run first.
```suggestion
MetricReg="${42}"

case "$MetricReg" in
    (NEW|OLD|BOTH)
        ;;
    (*)
        log_Err_Abort "unrecognized thickness compensation method: '$MetricReg', use NEW, OLD, or BOTH"
        ;;
esac
```